packages/rum-core/src/domain/action/getActionNameFromElement.ts

```diff
@@ -208,6 +213,15 @@ function getElementById(refElement: Element, id: string) {
   return refElement.ownerDocument ? refElement.ownerDocument.getElementById(id) : null
 }
 
+function getActionNameFromTextualContent(
+  element: Element | HTMLElement,
+  userProgrammaticAttribute: string | undefined,
+  privacyEnabledActionName?: boolean
+): ActionName | undefined {
+  const name = getTextualContent(element, userProgrammaticAttribute, privacyEnabledActionName)
+  return name ? { name, nameSource: ActionNameSource.TEXT_CONTENT } : undefined
```

Comment:
**💬 suggestion:** ‏I don't think returning `undefined` is useful here, since we'll fallback when the name is empty anyway
```suggestion
  return {
    name: getTextualContent(element, userProgrammaticAttribute, privacyEnabledActionName),
    nameSource: ActionNameSource.TEXT_CONTENT
  }
```

Reply:
This was for the cases when we fallback to `blank` if there is no `text_content`, or we can directly return `text_content` or `blank` here. Wdyt?